src/mrnet/core/rates.py

```diff
@@ -341,6 +323,8 @@ def __init__(self, reactants, products, ea_reference, delta_h_reference, alpha=0
         self.alpha = alpha
 
         super().__init__(reactants, products, None)
+        self.pro_mols = [p.mol_graph.molecule for p in products]
+        self.rct_mols = [r.mol_graph.molecule for r in reactants]
```

Comment:
Do we actually need the molecules for any reason? I'd say no. This is a rate calculator. Only the thermo is relevant. If you want to store the entry ids or something for bookkeeping purposes, I'd find that more agreeable.

Reply:
Addressed in recent commit.